src/Pillar-ExporterCore.package/PRDocumentListTransformer.class/instance/sort.limit.by..st

```diff
@@ -0,0 +1,7 @@
+sorting
+sort: aCollectionOfReferences limit: anInteger by: aString
+	| references |
+	references := aCollectionOfReferences first: anInteger.
+	aString = 'date' ifTrue: [ ^ PRDocumentListDateSorter with: references ].
+	aString = 'name' ifTrue: [ ^ PRDocumentListNameSorter with: references ].
```

Comment:
We will have to see how to sort by default, and what happens if there is an invalid value.